src/cryptoadvance/specter/templates/base.jinja

```diff
@@ -327,6 +327,7 @@
 			{% endif %}
 		</script>
 	{% endif %}
+	<!-- This scripts block is boing overwritten in /device -->
```

Comment:
boing?

Reply:
should be: `	<!-- This scripts block will be overwritten in /device -->`

Reply:
Done in  5930358e